sync_diff_inspector/source/chunks_iter.go

```diff
@@ -31,31 +31,33 @@ type ChunksIterator struct {
 	ID            *chunk.ChunkID
 	tableAnalyzer TableAnalyzer
 
-	TableDiffs     []*common.TableDiff
-	nextTableIndex int
-	chunksCh       chan *splitter.RangeInfo
-	errCh          chan error
-	limit          int
+	TableDiffs       []*common.TableDiff
+	nextTableIndex   int
+	chunksCh         chan *splitter.RangeInfo
+	errCh            chan error
+	limit            int
+	splitThreadCount int
 
 	cancel context.CancelFunc
 }
 
-func NewChunksIterator(ctx context.Context, analyzer TableAnalyzer, tableDiffs []*common.TableDiff, startRange *splitter.RangeInfo) (*ChunksIterator, error) {
+func NewChunksIterator(ctx context.Context, analyzer TableAnalyzer, tableDiffs []*common.TableDiff, startRange *splitter.RangeInfo, splitThreadCount int) (*ChunksIterator, error) {
 	ctxx, cancel := context.WithCancel(ctx)
 	iter := &ChunksIterator{
-		tableAnalyzer: analyzer,
-		TableDiffs:    tableDiffs,
-		chunksCh:      make(chan *splitter.RangeInfo, 64),
-		errCh:         make(chan error, len(tableDiffs)),
-		cancel:        cancel,
+		splitThreadCount: splitThreadCount,
+		tableAnalyzer:    analyzer,
+		TableDiffs:       tableDiffs,
+		chunksCh:         make(chan *splitter.RangeInfo, 30*splitThreadCount),
```

Comment:
could you add some comments about magic number 30?

Reply:
done